pkgs/development/python-modules/sphinxawesome-theme/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  poetry-core,
+  sphinx,
+  beautifulsoup4
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxawesome_theme";
+  version = "5.1.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-OwikuKJrPo4vNaud/9JToYYJePV6Kew8izYbr/qKTtQ=";
+  };
+
+  prePatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail "^7.2,<7.3" ">=7.3"
+  '';
```

Comment:
Don't use that, use `pythonRelaxDepsHook`, it has been made for that.

Reply:
I didn't knew that existed :o

Cant make it work though, may you help me on this one?

Reply:
Here's a very recent example: https://github.com/NixOS/nixpkgs/pull/315400/files#diff-83b3b91e3066d434e86c608d2f57d37c5cc5d806c433728b39e9de106f23ef50

Reply:
Does this help?

Reply:
perfect!